lite/examples/image_segmentation/android/build.gradle

```diff
@@ -6,9 +6,9 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
-        classpath 'de.undercouch:gradle-download-task:4.0.0'
-        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.31'
+        classpath 'com.android.tools.build:gradle:3.6.2'
```

Comment:
The Gradle plugin version 3.6+ has a specific requirement of NDK version even if the app doesn't use NDK. This breaks our internal CI. Please use `'com.android.tools.build:gradle:3.5.3'`

Reply:
Done, thanks for the heads up!